main.py

```diff
@@ -232,12 +232,17 @@ class Route:
       schema_migration.WriteUpdatedField),
 ]
 
-
+dev_routes: list[Route] = []
+if settings.DEV_MODE:
+  dev_routes = [
+    Route('/dev/clear_entities', dev_api.ClearEntities),
```

Comment:
I feel that this needs an extra layer of protection.  The settings.DEV_MODE and and your new settings.LOCAL_MODE depend on certain strings in the environment variables.  I can imagine that over time, there could be a variation in those strings... which would put the whole app in to DEV mode, but it might not be noticed at first.

How about commenting out these lines and having a comment saying that developers can uncomment them when needed?

Reply:
Alternatively, what about writing these as scripts under the scripts/ directory rather than being part of the app?

Reply:
> Alternatively, what about writing these as scripts under the scripts/ directory rather than being part of the app?

Is there a good way to do this without having the local server running? I tried writing the script and then realized it would not be of much use in its current form.

> How about commenting out these lines and having a comment saying that developers can uncomment them when needed?

I went ahead with this approach for now.

Reply:
> > Alternatively, what about writing these as scripts under the scripts/ directory rather than being part of the app?
> 
> Is there a good way to do this without having the local server running? I tried writing the script and then realized it would not be of much use in its current form.

Good point.  I don't think that it wold need to have the flask app running, but it would need the middleware for Cloud NDB.  And, it would need to do the steps for starting the cloud NDB emulator and stopping it after.  So, we could explore it sometime, but no need to do it now.